crates/bevy_a11y/README.md

```diff
@@ -0,0 +1,5 @@
+# Bevy A11Y
+
+[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y) is a component of the [Bevy Engine](https://bevyengine.org/). You can revise the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).
```

Comment:
so... `bevy` isn't an official crate?

Reply:
I could change it to:
[Bevy A11Y](https://github.com/bevyengine/bevy/tree/main/crates/bevy_a11y) is a component of the [Bevy](https://crates.io/crates/bevy) crate. You can review the list of official crates [here](https://github.com/bevyengine/bevy/tree/main/crates).

that would be better?

Reply:
I like that TBH